sled-agent/src/bootstrap/params.rs

```diff
@@ -41,6 +42,7 @@ struct UnvalidatedRackInitializeRequest {
     external_certificates: Vec<Certificate>,
     recovery_silo: RecoverySiloConfig,
     rack_network_config: RackNetworkConfig,
+    allowed_source_ips: AllowedSourceIps,
```

Comment:
Ahh, I think maybe this needs to regenerate an openAPI spec (`openapi/bootstrap-agent.json`), and once that happens then wicketd will fail to compile until it's updated to provide this new field.

Reply:
Oh, good point. I thought I hit these OpenAPI docs, but I probably just managed to update Nexus's. Will tackle this shortly.

Reply:
Welp, your hunch was right, and now there's a pile of wicket stuff to update :)